client/my-sites/stats/all-time/index.jsx

```diff
@@ -36,16 +36,25 @@ module.exports = React.createClass( {
 
 	render: function() {
 		var bestDay = null,
+			statTabs = [ 'posts', 'views', 'visitors', 'best' ],
+			classSets = {},
 			infoIcon = this.state.showInfo ? 'info' : 'info-outline',
 			valueClass,
 			bestViews,
 			classes;
 
+		statTabs.forEach( function( tabName ) {
+			var tabClassOptions = {};
+			tabClassOptions[ 'module-tab' ] = true;
+			tabClassOptions[ 'is-' + tabName ] = true;
+			classSets[ tabName ] = classNames( tabClassOptions );
+		} );
```

Comment:
here's one way to do it with the immutable functions

```js
import compose from 'lodash/function/compose';

const classesForTab = name => ( { 'module-tab': true, `is-${ name }`: true } );
const mergeByKeyName = op = ( obj, key ) => Object.assign( obj, op( key ) );

const classSets = 
	[ 'posts', 'views', 'visitors', 'best' ]
	.reduce( mergeByKeyName( compose( classNames, classesForTab ) ), {} );
```

haha, don't worry if this looks foreign or strange. it's here for fun and for reference if you have interest.